elements/pfe-tabs/demo/index.html

```diff
@@ -101,8 +101,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Horizontal wind</h2>
-      <pfe-tabs pfe-variant="wind">
+      <h2>Style: Horizontal wind with center alignment</h2>
+      <pfe-tabs pfe-align="center" pfe-variant="wind">
```

Comment:
What would you think of the attribute name `pfe-tab-align` instead?  To clarify that the alignment relates to the tabs and not the content for example?

Reply:
@starryeyez024 bump ^

Reply:
works for me :)

Reply:
updated